### PR TITLE
Fix logic for detecting if Python2 is supported

### PR DESCRIPTION
**Pull request by e4t**
This fixes Issue #445.
Thanks to Matt Ezell from  Oak Ridge.

Signed-off-by: Egbert Eich <eich@suse.com>

### CODE REVIEW

File: clustershell.spec.in [hunk 1]
```diff
@@ -21,7 +21,7 @@
 %{!?__python3: %global __python3 python3}
 %{!?python3_shortver: %global python3_shortver %(%{__python3} -c 'import sys; print(str(sys.version_info.major) + "." + str(sys.version_info.minor))')}
 
-%if ! 0%{?rhel} >= 8 && ! 0%{?suse_version} > 1550
+%if 0%{?rhel} < 8 &&  0%{?suse_version} <= 1500
```
Comment (martinetd) [inline clustershell.spec.in:24]: Matt in the issue only inverted the equality, but you also changed the suse version from 1550 to 1500 -- is that on purpose?
(might want to say that in the commit message if so)

Comment (mattaezell) [reply clustershell.spec.in:24]: It was unintentional - I don't have OpenSuSE so I'm not sure exactly what version stopped supporting py2. Changing the 1500 back to 1550 is probably the right thing to do.

Comment (e4t) [reply clustershell.spec.in:24]: %suse_version 1500 includes Leap/SLE 15 - which still has python2. %suse_version 1550 is Tumbleweed/Factory (currently) where python2 has been dropped from.

Comment (martinetd) [reply clustershell.spec.in:24]: Ok, so from that I take we don't want python2 in Tumbleweed (1550), so the new logic is correct -- wasn't that just what the problem is about?

Well, non-convoluted logic is better for everyone anyway, thanks!

**Review by martinetd — APPROVED** [review]

**Review by mattaezell — APPROVED** [review]

**Comment by degremont** [comment]
@e4t the test failed for unrelated reason. Refreshing the commit would retrigger them. It could be good to update your commit message to follow the way we usually do them. See as an example: https://github.com/cea-hpc/clustershell/commit/23621a5f42ddca7d50c7122f83153e764f314b74#diff-8798411bf1e861edaa6a04787c30ade5

Basicly (feel free to adapt or reword): 

    specfile: fix logic for detecting if Python2 is supported
   
    Python2 was conditionally enabled only if building for 
    an old RHEL or old SLES but the version tests was not correct.

    Thanks to Matt Ezell from Oak Ridge.

    Fixes #445.
    Signed-off-by: Egbert Eich eich@suse.com

**Comment by e4t** [comment]
I've updated and pushed it again, now.

**Comment by thiell** [comment]
Hi @e4t, thanks for the PR! LGTM.

Could you please push your last [change](https://github.com/e4t/clustershell/commit/6accec491f0fa3cbeb8ff3321fb554272c7e08bd) to the [e4t/master](https://github.com/e4t/clustershell/tree/e4t/master) branch so this PR gets updated?

**Comment by e4t** [comment]
Done. I had accidentally created an additional branch.

**Comment by thiell** [comment]
Thanks @e4t!

**Pull request merged**